src/Weather/Skysight/SkySightRequest.cpp

```
@@ -0,0 +1,996 @@
+}
+
+static_assert(ShouldRetryHttpDownload(429, 99));
+static_assert(ShouldRetryHttpDownload(503, 0));
+static_assert(!ShouldRetryHttpDownload(503, 2));
+static_assert(!ShouldRetryHttpDownload(404, 0));
+
+static boost::json::value
+ParseJsonResponse(std::string_view body, const char *context)
+{
+  boost::system::error_code ec;
+  auto value = boost::json::parse(body, ec);
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)